### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by aynader** [AbyszOne/Abysz-LAB-Ext#5]
I have been trying out your extension for a while and it looks quite promising! I have had mixed results so far but overall its robust albeit the user experience could be better. But it is sometimes frustrating, I can't tell how many times I have broken it and had to restart the whole UI because I did something too fast or clicked twice accidentally, I suspect adding some more try-excepts will solve the problem but I am not quite sure because some of the code is in spanish?

**Comment by AbyszOne** [comment]
Yes, the notes are in Spanish, it is my language. About the GUI, it can probably get bugs, but honestly I don't know of a case where it "breaks". It does not use dependencies from the main GUI, so each process can be rerun even if it fails.
If you have examples, I'll try to improve it.

**Comment by aynader** [comment]
I have no specific example as I was unable to reliably reproduce the "breaking" but at least here's something I am sure of:
If a folder exists within the directory of images when deflickering, it bricks it, while this is not necessarily a bug, it's inconvenient. 
Overall awesome work!

**Comment by AbyszOne** [comment]
Thank you.
That's right. Although it is expected to use individual folders for image sequences, there is such a conflict. I'll see to correct it in updates. There are several pending improvements, but I have not spent time these weeks, since there are no queries. Feel free to ask for improvements.
